test_vm/src/lib.rs

```diff
@@ -571,6 +578,13 @@ pub struct InvocationCtx<'invocation, 'bs> {
     caller_validated: bool,
     policy: &'invocation Policy,
     subinvocations: RefCell<Vec<InvocationTrace>>,
+    actor_exit: RefCell<Option<ActorExit>>,
```

Comment:
Please document this - it's not obvious what it is, when it's set, why it's needed, etc. It should probably move above the subinvocations too.